dokku

```diff
@@ -132,7 +132,11 @@ case "$1" in
 
         # if we can't post-deploy successfully, kill new container
         kill_new() {
-          docker inspect $id &> /dev/null && docker stop $id > /dev/null && docker kill $id > /dev/null
+          docker inspect $id &> /dev/null \
+          && docker stop $id > /dev/null \
+          && docker kill $id > /dev/null \
+          && docker rm $id > /dev/null
```

Comment:
Unfortunately this prevents a user from seeing the details of why their deployment failed.